keep/api/core/db.py

```diff
@@ -2585,6 +2585,24 @@ def update_incident_summary(tenant_id: str, incident_id: UUID, summary: str) ->
         session.refresh(incident)
         
         return 
+    
+def update_incident_name(tenant_id: str, incident_id: UUID, name: str) -> Incident:
+    if not name:
```

Comment:
how could it be? sounds like an error/warning. why would someone update_incideint_name without a name?

Reply:
I would at least add warning log for that